src/taxinyc/data_processor.py

```
@@ -0,0 +1,87 @@
+from sklearn.impute import SimpleImputer
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+class DataProcessor:
+    def __init__(self, sparksession, table, config):
+        self.df = self.load_data(sparksession, table)
+        self.config = config
+        self.X = None
+        self.y = None
+        self.preprocessor = None
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)